src/client.rs

```diff
@@ -524,6 +515,11 @@ impl<'mpesa, Env: ApiEnvironment> Mpesa<Env> {
     }
 }
 
+#[derive(Debug, serde::Deserialize)]
+struct AuthResponse {
```

Comment:
You are missing an extra field here which is `expiry_in`, if you want to ignore this value, derive serde default as well

Reply:
I had actually included it but I realized I was not using the value anyway, at least in this PR. You can include it in #78 if you will need to reference it there

Reply:
WDYT?

Reply:
I just checked out #78, I think I could omit my changes in favor of yours just to avoid any mix up